style.css

```diff
@@ -3,10 +3,77 @@
 }
 
 .calculator-container {
-  border: solid;
-  border-width: 1px;
-  height: 400px;
-  width: 300px;
-  border-radius: 15px;
+  width: 40rem;
+  height: 50rem;
   margin: auto;
+  border: solid;
+  border-radius: 1rem;
+}
+
+.screen {
+  grid-column: 1 / 5;
+  border-width: 0.5px;
+  border-top-left-radius: 0.7rem;
+  border-top-right-radius: 0.7rem;
+  background-color:rgb(42, 21, 65)
+}
+
+.keys-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 20% repeat(5, 1fr);
+  height: 100%;
+
+}
+
+.key {
+  display: flex;
+  display: -webkit-flex;
+  border-width: 0.5px;
+  font-size: 3rem;
+  border: solid; 
+  border-color: black;
+  color: white;
```

Comment:
We use the colors in Hexadecimal, thank you. 

Valid for all background color also ^^